packages/components/app/styles/components/side-nav/toggle-button.scss

```diff
@@ -16,7 +16,7 @@
   display: flex;
   flex-direction: row-reverse;
   align-items: center;
-  width: 26px;
+  width: 24px;
```

Comment:
general question (not a blocker): since it's only a single entry point, should we convert it to a CSS variable so that we can keep it as is now (_**if**_ this is by design specs) and at the same time allow consumers to override it (via CSS variable) in "special" cases?

Reply:
exposed the button width as a CSS variable in c1666b187c7d8d47a5e0f7933730cfab51d7f92b – let me know what you think